src/pyairios/models/vmn_05lm02.py

```diff
@@ -16,19 +19,46 @@
     U16Register,
 )
 
+LOGGER = logging.getLogger(__name__)
+
 
 class Reg(RegisterAddress):
     """Register set for VMN-05LM02 remote node."""
 
     REQUESTED_VENTILATION_SPEED = 41000
 
 
-class VMN05LM02(AiriosDevice):
-    """Represents a VMN-05LM02 remote node."""
+class NodeData(AiriosDeviceData):
+    """VMN-05LM02 remote node data."""
+
+    requested_ventilation_speed: Result[VMDRequestedVentilationSpeed] | None
+
+
+def pr_id() -> int:
+    """
+    Get product_id for model VMN_05LM02.
+    Named as is to discern from node.product_id register.
+    :return: unique int
+    """
+    return 0x0001C83E
+
+
+def product_descr() -> str | tuple[str, ...]:
+    """
+    Get description of product(s) using VMN_05LM02.
+    Human-readable text, used in e.g. HomeAssistant Binding UI.
+    :return: string or tuple of strings, starting with manufacturer
+    """
+    return "Siber 4 button Remote"
+
+
+class Node(AiriosDevice):
```

Comment:
I don't like having the same class name in different files. I guess it is to have a common entry point to instantiate without knowing the final class name.

I think we need to implement a factory pattern, E.g. https://medium.com/@amirm.lavasani/design-patterns-in-python-factory-method-1882d9a06cb4

Each supported model file could have a static variable "factory":
`factory = VMN05LM02Factory()`

And then in `do_node()` do `node_module = brdg_data["models"][key].factory.create_node()`

Reply:
Good idea with implementation, thanks. Haven't built the `(abstract_)factory` yet.